tests/cartesian_coordinates/xyz_functions/test_xyz_functions.py

```diff
@@ -74,3 +74,27 @@ def test_concat_with_zmats():
         ),
         znew.get_cartesian(),
     )
+
+
+def test_multiple_xyz():
+    output_path = os.path.join(STRUCTURES, "xyz_test.xyz")
+
+    path1 = os.path.join(STRUCTURES, "MIL53_small.xyz")
+    path2 = os.path.join(STRUCTURES, "nasty_cube.xyz")
+    path3 = os.path.join(STRUCTURES, "MeOH_Furan_end.xyz")
+    molecule1 = cc.Cartesian.read_xyz(path1)
+    molecule2 = cc.Cartesian.read_xyz(path2)
+    molecule3 = cc.Cartesian.read_xyz(path3)
+
+    cartesian_list = [molecule1, molecule2, molecule3]
+
+    cc._cartesian_coordinates.xyz_functions.multiple_to_xyz(cartesian_list, output_path)
```

Comment:
Can you expose these to the user-level API, i.e. import them in the `chemcoord.xyz_functions` such that users (and yourself in this test) can just import `cc.xyz_functions.multiple_to_xyz`?

Can you additionally add stub rst files, such that these functions get a documentation as for example

https://chemcoord.readthedocs.io/en/v2.1.2/src_xyz_functions/chemcoord.xyz_functions.to_molden.html